tools/system_libs.py

```diff
@@ -2314,6 +2322,7 @@ def add_sanitizer_libs():
     add_library('libsockets')
 
   if settings.USE_WEBGPU:
+    add_library('libwebgpu')
     add_library('libwebgpu_cpp')
```

Comment:
Unrelated.. but we should probably gate this second library on `settings.LINK_AS_CXX`